Fleece/API_Impl/Fleece.cc

```diff
@@ -229,6 +228,31 @@ FLMutableArray FLMutableArray_New(void) FLAPI {
     return _newMutableArray(nullptr, kFLDefaultCopy);
 }
 
+FLMutableArray FLMutableArray_NewFromJSON(FLString json, FLError* outError) FLAPI {
+    if (outError != nullptr) {
+        *outError = kFLNoError;
+    }
```

Comment:
FYI, these lines aren't necessary -- in our calling conventions, if the function returns a success value, the contents of `*outError` are explicitly undefined. (You don't need to take them out, just letting you know.)